logstash-core-event/spec/logstash/timestamp_spec.rb

```diff
@@ -81,4 +81,29 @@
       expect(t - 10).to eq(current)
     end
   end
+
+  context "identity methods" do
+    subject { LogStash::Timestamp.new }
+
+    it "should support utc" do
+      expect(subject.utc).to eq(subject)
```

Comment:
The Timestamp contract is that any time will be converted in UTC so the `utc` and `gmtime` methods are just identity function because a `Timestamp` is always UTC.